powerbi-docs/transform-model/datamarts/datamarts-access-control.md

```diff
@@ -90,6 +90,7 @@ A datamart can get marked as an unavailable datamart when one of the following s
 
 :::image type="content" source="media/datamarts-access-control/datamarts-access-control-01.png" alt-text="Screenshot of the request access setting.":::
 
+**Situation 3:** When a Premium workspace is migrated to another Premium Capacity in a different region, the datamart will become unavailable with the error "Unable to open the datamart because the workspace region has changed. To open the datamart, reconnect the workspace to the region connected when the datamart was created." This behavior is by design, since the region where the datamarts were created must be the region where the workspace resides, and migrations are not supported.
```

Comment:
```suggestion
**Situation 3:** When a Premium workspace is migrated to another Premium capacity in a different region, the datamart will become unavailable with the error: "Unable to open the datamart because the workspace region has changed. To open the datamart, reconnect the workspace to the region connected when the datamart was created." This behavior is by design since the region where the datamarts were created must be the region where the workspace resides, and migrations are not supported.
```